go.mod

```
@@ -1,18 +1,19 @@
 module github.com/burik666/gobin
 
-go 1.17
+go 1.22.0
+
+toolchain go1.23.3
```

`toolchain` directive is not needed here.

---

do you mean upgrade go line to go 1.23.3?

---

I mean that the toolchain directive should be removed.
```suggestion
go 1.22.0
```

---

ok